src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarSizeInfo.tsx

```diff
@@ -11,16 +11,22 @@ export interface ArtworkSidebarSizeInfoProps {
 export class ArtworkSidebarSizeInfo extends Component<
   ArtworkSidebarSizeInfoProps
 > {
+  sizeInfoMissing = (dimensions, edition_of) => {
+    const dimensionsPresent =
+      /\d/.test(dimensions?.in) || /\d/.test(dimensions?.cm)
+
+    return !(edition_of && edition_of.length) && !dimensionsPresent
```

Comment:
Note you can use the existential operator to slightly simplify:

```ts
return !edition_of?.length && !dimensionsPresent
```

Reply:
Ooh, good call.

Pushed!